architecture/core_concepts/routes.adoc

```diff
@@ -352,6 +354,118 @@ and deleted, the router adds and removes corresponding pool members.
 - When a user deletes the route and all endpoints associated with a particular
 pool, the router deletes that pool.
 
+[[architecture-f5-native-integration]]
+==== F5 Native Integration
+
+With new
```

Comment:
If by "new" this means it's new in this version, I'd shy away from it. I can't imagine this would get picked up in the next release or whatever (but if you plan to delete it, I take it back). I think if just the word "new" went away, this would mean the same.